geonode/contrib/monitoring/collector.py

```diff
@@ -130,7 +131,8 @@ def get_mem_label(*args):
                               )
                              )
 
-        collected_at = datetime.now()
+        utc = pytz.utc
+        collected_at = datetime.utcnow().replace(tzinfo=utc)
```

Comment:
Other way is to use `tzinfo.localize`, so in this case, `collected_at = utc.localize(datetime.now())`